package/MDAnalysis/lib/src/transformations/transformations.c

```diff
@@ -786,7 +786,7 @@ PyConverter_AnyDoubleArray(
     PyObject **address)
 {
     PyArrayObject *obj = (PyArrayObject *)object;
-    if (PyArray_Check(object) && obj->descr->type_num == PyArray_DOUBLE) {
+    if (PyArray_Check(object) && obj->descr->type_num == NPY_DOUBLE) {
```

Comment:
@hmacdope or @richardjgowers - am I correct in thinking that this is an acceptable replacement for the now removed PyArray_DOUBLE?

Reply:
Looks correct to me, there is even a `tools/replace_old_macros.sed` in the NumPy repo that does what you did.